src/pages/lib-v1/storage/transfer-acceleration/q/platform/[platform].mdx

```diff
@@ -0,0 +1,16 @@
+export const meta = {
+  title: `Use Transfer Acceleration`,
+  description: `How to enable and use S3's Transfer Acceleration on your S3 bucket`,
+};
+
+import js from "/src/fragments/lib-v1/storage/js/transfer-acceleration.mdx";
+
+<Fragments fragments={{js: js}} />
+
+import androidDoc from '/src/fragments/lib/storage/native_common/transfer-acceleration/common.mdx';
+
+<Fragments fragments={{ android: androidDoc }} />
+
+import flutterDoc from '/src/fragments/lib/storage/native_common/transfer-acceleration/common.mdx';
+
+<Fragments fragments={{ flutter: flutterDoc }} />
```

Comment:
Ditto, I think we should avoid adding new content to other platform's `lib-v1` routes

Reply:
Took an [existing reference for this as well from V5](https://github.com/aws-amplify/docs/blob/main/src/pages/lib/storage/transfer-acceleration/q/platform/%5Bplatform%5D.mdx)

Reply:
ahh makes sense, updated